plugins/obs-transitions/transition-luma-wipe.c

```diff
@@ -35,11 +39,16 @@ static const char *luma_wipe_get_name(void *type_data)
 
 static void luma_wipe_update(void *data, obs_data_t *settings)
 {
-	struct luma_wipe_info *lwipe = data;
+	struct luma_wipe_info *lwipe = data;	
```

Comment:
Style:  Don't add trailing whitespace.

```suggestion
	struct luma_wipe_info *lwipe = data;
```